model/src/main/proto/arguments.proto

```diff
@@ -29,3 +29,22 @@ enum HighlightItem {
   // Indicates that the Developer Options item needs to be highlighted.
   DEVELOPER_OPTIONS_ITEM = 4;
 }
+
+// Arguments needed to open the policy page.
+message PoliciesArguments {
+  // The specific policy page that should be displayed.
+  PolicyPage policy_page = 1;
+}
+
+// Represents the type of policy page.
+enum PolicyPage {
```

Comment:
I suggest nesting this within the message above since it should only ever be used in relation to the PolicyActivity.

Reply:
Done.